index.html

```diff
@@ -4853,6 +4853,72 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to be usable to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking proofs on <a>verifiable presentations</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by appropriate entities.  Validation is the process by which verifiers apply business rules to 
+evaluate the appropriateness of a particular use of a Verifiable Credential.
+        </p>
+        <ul>
+          <li>
+<a href="#issuer">Issuers</a> define expressions of `issuer` in <a>credentials</a>
+          </li>
+          <li>
+<a href="#presentations-0">Presentations</a> define expressions of `holder` in <a>presentations</a>
+          </li>
+          <li>
+<a href="#credential-subject">Credential subjects</a> define expressions of `credentialSubject` in <a>credentials</a>
+          </li>
+        </ul>
+        <p>
+A <a>verifier</a> might need to validate a given <a>verifiable presentation</a>
```

Comment:
```suggestion
A <a>verifier</a> might need to evaluate a given <a>verifiable presentation</a>
```

Reply:
Are these 2 words actually the same, or does validate have specific meaning in the data model, that is different than evaluate?

Reply:
Are these 2 words actually the same, or does validate have specific meaning in the data model, that is different than evaluate?

Reply:
I suggested this change based off of your suggestion here: https://github.com/w3c/vc-data-model/pull/1199#issuecomment-1644407950

If you didn't mean to imply that we should use the word "evaluate", then you don't need to take this change.

The word "validate" does have a specific meaning in the data model: 

https://w3c.github.io/vc-data-model/#dfn-credential-validation

We have a whole section on it in the spec:

https://w3c.github.io/vc-data-model/#validation

Reply:
Sorry if my comment there was not clear, I was trying to say, we should use the word validate if it's meaning in our spec matches it's use here, and avoid using a new term.

If the words are essentially synonyms we should update our terminology to express that, especially to assist with internationalization.

Until that happens I prefer to keep it as.

Reply:
@OR13 wrote:
> Until that happens I prefer to keep it as.

Works for me, resolving conversation.